pkg/acs/calacs/acssum/mainsum.c

```diff
@@ -51,20 +52,30 @@ int main (int argc, char **argv) {
 	/* Get names of input and output files. */
 	for (i = 1;  i < argc;  i++) {
 	    if (argv[i][0] == '-') {
-		for (j = 1;  argv[i][j] != '\0';  j++) {
-		    if (argv[i][j] == 't') {
-			printtime = YES;
-		    } else if (argv[i][j] == 'v') {
-			verbose = YES;
-		    } else if (argv[i][j] == 'q') {
-			quiet = YES;
-		    } else {
-				printf (MsgText, "Unrecognized option %s\n", argv[i]);
-				free (input);
-				free (output);
-				exit (1);
-		    }
-		}
+            if (!(strcmp(argv[i],"--version")))
```

Comment:
Awesome, cheers for spotting this. Changes updated. BTW I would branch this such that when I squash everything you can pull the squashed version, and do a simple diff as your final review before I merge.